DDIG/SynthStrip/1.0.0/weights/synthstrip.1.pt

```diff
@@ -0,0 +1 @@
+../.git/annex/objects/JG/7x/SHA256E-s30851709--37417f802196186441aae3e7f385d94f8a98c64a88acaeaa2723af995c653e33.1.pt/SHA256E-s30851709--37417f802196186441aae3e7f385d94f8a98c64a88acaeaa2723af995c653e33.1.pt
```

Comment:
I can see the model file is saved in annex here but this file was not available in OSF so, probably it is not probably added or uploaded.

Reply:
@Hoda1394 - he won't have permission to add to OSF, but perhaps he could post url's here that you could add to the annex.

Reply:
what's important though is that it is git-annexed to some public URL.

Reply:
That sounds good, I will try to see If I can do it with the github action.